src/tools/r2rdump/R2RReader.cs

```diff
@@ -364,6 +364,9 @@ private void ParseAvailableTypes()
             {
                 return;
             }
+
+            Dictionary<uint, bool> added = new Dictionary<uint, bool>();
```

Comment:
If you just want to quickly determine existence, a `HashSet<uint>` would be sufficient